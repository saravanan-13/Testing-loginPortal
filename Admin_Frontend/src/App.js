import React from 'react';
import logo from './logo.svg';
import './App.css';
import Admin from './components/admin';

function App() {
  return (
    <div>
      <Admin />
    </div>
  );
}

const styleLink = document.createElement("link");
styleLink.rel = "stylesheet";
styleLink.href = "https://cdn.jsdelivr.net/npm/semantic-ui/dist/semantic.min.css";
document.head.appendChild(styleLink);

export default App;

