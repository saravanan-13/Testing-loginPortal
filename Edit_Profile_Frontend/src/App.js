import React from 'react';
import logo from './logo.svg';
import './App.css';
import EditProfile from './components/edit_profile';

function App() {
  return (
   
    <div>
      <EditProfile/>
    </div>
  );
}

export default App;
