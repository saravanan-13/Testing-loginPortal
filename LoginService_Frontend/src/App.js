import React, {Component} from 'react';

import './App.css';
import Forms from './components/Forms'
// import {Button , Form , FormGroup, label, Input, FormFeedback}
// from 'reactstrap';
//import Axios from 'axios';
//import ReactDOM from 'react-dom';



class App extends Component {

  state={
    userData: []
  }
 
 render()
 {
   return (
    
      <Forms />

   )
 }

}

export default App;
