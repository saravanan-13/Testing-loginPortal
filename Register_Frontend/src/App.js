/**
  * 1990-2019 Publicis Sapient Corporation. All rights reserved.   
*/

import React from 'react';
import './App.css';
import RegisterButton from './components/RegisterButton';


function App() {
  return (
    <div className="App">
      <RegisterButton />
    </div>
  );
}

export default App;
