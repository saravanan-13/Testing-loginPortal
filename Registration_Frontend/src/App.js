import React from 'react';
import './App.css';
import Main from './components/Main';

function App() {
  return (
    <div className="App">
      <h1>Home</h1>
      <br/>
      <Main/>
    </div>
  );
}

export default App;
