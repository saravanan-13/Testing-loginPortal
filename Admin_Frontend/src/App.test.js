import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import Adapter from 'enzyme-adapter-react-16';
import Admin from './components/admin';
import { mount, shallow, render,configure } from 'enzyme';
configure({adapter: new Adapter()});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});
