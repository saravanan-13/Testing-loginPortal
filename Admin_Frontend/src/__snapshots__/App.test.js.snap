// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be possible to activate button with Spacebar 1`] = `ReactWrapper {}`;
